napalm/base/helpers.py

```diff
@@ -155,12 +161,19 @@ def regex_find_txt(pattern, text, default=""):
     value = re.findall(pattern, text)
     try:
         if not value:
-            raise Exception
+            logger.error("No Regex match found for pattern: %s" % (str(pattern)))
```

Comment:
This feels a bit redundant.

Reply:
I hear ya that it feels redundant but the main reason I needed to open the PR was that depending on the driver implementation, these kinds of errors do get passed over when the get raised. For example if a driver implementation passes on a try/except, this will catch the condition globally. While I'd prefer if those kinds of situations were handled at and audited when a driver PR is made & committed, this is a "rather safe than sorry" implementation ;-)